docs/app/pages/components/components-sections/tables/column-picker/column-picker.component.html

```diff
@@ -79,6 +86,24 @@
     </tr>
 </uxd-api-properties>
 
+<uxd-api-properties tableTitle="ColumnPickerGroupItem Interface">
+    <tr uxd-api-property name="group" type="string">
+        The name of the group that this column belongs to.
+    </tr>
+    <tr uxd-api-property name="name" type="string">
+        The name of the column.
+    </tr>
+</uxd-api-properties>
+
+<uxd-api-properties tableTitle="ColumnPickerGroupSetting Interface">
+    <tr uxd-api-property name="group" type="string">
+        The name of the group this setting object is related to.
+    </tr>
+    <tr uxd-api-property name="initiallyExpanded?" type="boolean">
```

Comment:
The `?` isn't part of the property name. I think the proper way of denoting this would be to add `defaultValue="false"`.